packages/code-du-travail-frontend/src/outils/DureePreavisDemission/steps/__tests__/UnsupportedCCDisclaimer.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import UnsupportedCCDisclaimer from "../component/UnsupportedCCDisclaimer";
+
+describe("Unsupported CC Disclaimer component should render disclaimer", () => {
+  it("with no link if no id", () => {
```

Comment:
> with no link if no id

Ce n'est pas très parlant. Je n'ai pas compris ce que l'on entendait par link et id ici.

Proposition:

```suggestion
  it("should not show the link if agreement is not available on legifrance", () => {
```